src/main/kotlin/entry/dsm/gitauth/equusgithubauth/domain/user/entity/repository/UserRepository.kt

```diff
@@ -0,0 +1,8 @@
+package entry.dsm.gitauth.equusgithubauth.domain.user.entity.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import entry.dsm.gitauth.equusgithubauth.domain.user.entity.User
+
+interface UserRepository : JpaRepository<User, Long> {
```

Comment:
```suggestion
interface UserRepository : JpaRepository<User, UUID> {
```
JpaRepository에서 기본 키 타입을 잘못 지정하면 런타임 에러가 발생할 가능성이 있습니다. 예를 들어, findById 메서드를 호출할 때 엔터티의 실제 PK 타입이 불일치하면 에러가 발생할 수 있습니다.